### PR TITLE
Bdtt 12 varun backend

### PR DESCRIPTION
**Pull request by VaruneshBala**
# Description
Fixes [Jira ticket BDTT-12]()

BDTT-12 backend

List all acceptance criterias of user story and indicate which has been solved previously and which is solved currently
- [x] XXX (solved)
- [] XXX (solving)
- [ ]

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactor
- [ ] Migration

# Checklist:

- [X] I have added the appropriate labels to my PR
- [X] My code follows the style guidelines of this project
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by VaruneshBala** [comment]
can only return one container number so far need to find a way to return all

need to test viewShipments function bc i cant download invokes on my local machine

**Comment by VaruneshBala** [comment]
viewallshipments works now pls check
included all the core svcs on docker-compose
made some changes to port number in all the dockerfiles for core svcs for brayden (changed to port 80 and did port mapping in docker -compose)

**Review by prussianman — CHANGES_REQUESTED** [review]
Thanks for the laborious work, mustn't have been easy to test all the code. 

However, there are some issues:

1. There is a port conflict between 2 containers sharing port 8081.

2. The core APIs were to use ports 5000+ , while the scrapers port 8080+, however, they are both using 8080+ (there will be conflicts when merged)

3. The PROD dockerfiles are not PROD ready (this is fine, I will open another branch to work on them)

**Comment by prussianman** [comment]
LGTM

**Review by prussianman — APPROVED** [review]
LGTM

File: services/backend/core/export_cont.py [hunk 1]
```diff
@@ -55,5 +55,36 @@ def get_export_ref_n():
         }
     ), 500
 
+# Retrieve CONT_N from EXPORT_REF_CONT using EXPORT_REF_N (NOT ABLE TO GET ALL VALUES ONLY THE FIRST VALUE RETRIEVED)
```
[the remark this answers is not in the document]
Comment (VaruneshBala) [reply services/backend/core/export_cont.py:58]: Nope I can retrieve all now. Forgot to remove the comment

File: services/backend/core/export.py [hunk 2]
```diff
@@ -58,5 +61,36 @@ def get_export_ref_n():
         }
     ), 500
 
+# Retrieve EXPORT_REF_N using WGUSER_ID
+@app.route("/export/export_ref_n/wguser_id", methods=['POST'])
+def get_export_ref_n_using_wguser_id():
+    data = request.get_json()
+    wguser_id = data["wguser_id"]
+    export_ref_n = Export.query.filter_by(wguser_id=wguser_id).first().export_ref_n
```
[the remark this answers is not in the document]
Comment (VaruneshBala) [reply services/backend/core/export.py:69]: Oh so not one wguser_id = one import/export_ref_num?

[the remark this answers is not in the document]
Comment (VaruneshBala) [reply services/backend/core/export.py:69]: i will make changes tonight

File: services/backend/core/import.py [hunk 3]
```diff
@@ -21,17 +21,21 @@ class Import(db.Model):
...
         self.hbl_n = hbl_n
+        self.wguser_id = wguser_id
 
     def json(self):
         return {
             "import_ref_n": self.import_ref_n,
             "cust_id": self.cust_id,
-            "hbl_n": self.hbl_n
+            "hbl_n": self.hbl_n,
+            "uguser_id": self.wguser_id
```
[the remark this answers is not in the document]
Comment (VaruneshBala) [reply services/backend/core/import.py:37]: changed

File: services/backend/core/import.py [hunk 4]
```diff
@@ -58,5 +62,31 @@ def get_import_ref_n():
         }
     ), 500
 
+# Retrieve IMPORT_REF_N using WGUSER_ID
+@app.route("/import/import_ref_n/wguser_id", methods=['POST'])
+def get_import_ref_n_using_wguser_id():
+    data = request.get_json()
+    wguser_id = data['wguser_id']
+    import_ref_n = Import.query.filter_by(wguser_id=wguser_id).first().import_ref_n
```
[the remark this answers is not in the document]
Comment (VaruneshBala) [reply services/backend/core/import.py:70]: will change tonight

File: services/backend/core/viewAllShipments.py [hunk 5]
```diff
@@ -0,0 +1,85 @@
+# NOT ABLE TO INSTALL INVOKES ON MY END CAN SOMEONE HELP ME INSTALL AND TRY THIS OUT? all the other 4 services works individually
```
[the remark this answers is not in the document]
Comment (VaruneshBala) [reply services/backend/core/viewAllShipments.py:1]: Nope forgot to remove comment

**Comment by VaruneshBala** [comment]
Working on the necessary changes in a new branch (bdtt-12-varun-viewallshipment-BackEnd)

**Pull request merged**